### ISSUE TITLE
[stable/jenkins] install of ingress fails.

### ISSUE THREAD

**Issue by indpnday** [helm/charts#5541]
**Is this a request for help?**: NO

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):  BUG REPORT

**Version of Helm and Kubernetes**: Helm "v2.8.1"  - Kubernetes "v1.9.5"

**Which chart**: jenkins

**What happened**: install fails
```
helm --namespace jenkins --name jenkins -f ./values.yaml install stable/jenkins --debug
[debug] Created tunnel using local port: '35153'

[debug] SERVER: "127.0.0.1:35153"

[debug] Original chart version: ""
[debug] Fetched stable/jenkins to /root/.helm/cache/archive/jenkins-0.14.4.tgz

[debug] CHART PATH: /root/.helm/cache/archive/jenkins-0.14.4.tgz

Error: unable to recognize "": no matches for networking.k8s.io/, Kind=Ingress
```

**What you expected to happen**: install ok.

**How to reproduce it** (as minimally and precisely as possible): Define HostName and install.

**Anything else we need to know**: Applying this diff, it fixed:

```
# diff jenkins/templates/jenkins-master-ingress.yaml jenkins-fixed/templates/jenkins-master-ingress.yaml 
2c2
< apiVersion: {{ .Values.NetworkPolicy.ApiVersion }}
---
> apiVersion: {{ .Values.Master.Ingress.ApiVersion }}
21c21
< {{- end -}}
---
> {{- end }}
```

**Comment by edbizarro** [comment]
Could it be related ? https://github.com/kubernetes/charts/commit/c60086ec0b260996281d337d67f2361f826f7029#diff-ca9df7f2da6d1aba35009fa5756eedf8

**Comment by mattjmcnaughton** [comment]
I think this was fixed in [this commit](https://github.com/kubernetes/charts/commit/78ee4ba1546c0428756af2684d3c0eac82a3c04d#diff-e1d397f64becd8b83ea6469cd3cae80f). My guess is that the helm client has cached an old version that does not have this fix (as I can see its trying to deploy 0.14.4). If you run `helm repo update` you should be good to go :)

**Comment by indpnday** [comment]
You are right. The new version is fixed.
